content/src/content/jcr_root/apps/core/wcm/components/commons/editor/dialog/childreneditor/v1/childreneditor/clientlibs/js/childreneditor.js

```diff
@@ -252,25 +252,23 @@
                 }
 
                 Coral.commons.ready(that._elements.self, function(item) {
-                    // As a reordering of the multifield also triggers the coral-collection:remove event we have to add
```

Comment:
I think since that._elements.self is referenced multiple times, could probably be stored as a const and used in all the other places, but this is more a nitpick